torch/csrc/lazy/core/config.h

```diff
@@ -1,4 +1,7 @@
 #pragma once
 #include <c10/util/Flags.h>
 
+C10_DECLARE_int(torch_lazy_io_thread_pool_size);
```

Comment:
I will let @wconstab comment on these.

Reply:
Do you mean how to setup them through environment variables?

Reply:
I'm not sure what @alanwaketan means.

I made a bad assumption that all the declared flags would automatically be available as env vars or cmd line options.  I think this is true for gflags in some scenarios, but, i couldn't figure out how to set the env vars in torch so far.

So ultimately, i think we'll have to
 - convert some of these to pybindings and decide they don't need env vars at all
 - for ones that need env vars, somehow add support - either switch back to getenv, or, figure out how to turn on env parsing for C10_DECLARE flags.
 
 It probably doesn't hurt to keep following this pattern of moving all the envs into one config file though.  Might make it easier later on to make the above changes all-at-once?